File: problem01/Euler1.java
public class Euler1 {

	public Euler1() {
	}

	public static void main(String[] args) {
		int total = 0;
		for(int i = 0; i < 1000; i++) {
			if(isMultipleOfThree(i) || isMultipleOfFive(i)) {
				total = total + i;
			}
		}


		System.out.println(total);
	}

	private static boolean isMultipleOfThree(int n) {
		return n%3 == 0;
	}

	private static boolean isMultipleOfFive(int n) {
		return n%5 == 0;
	}

}

File: problem01/Euler1.py
def isMultipleOfThree(num):
    return num%3==0

def isMultipleOfFive(num):
    return num%5==0

total = 0
for x in range(0, 1000):
    if isMultipleOfThree(x) or isMultipleOfFive(x):
        total = total + x
        print total, x

print total
